ebusd-2.1.x/en/vaillant/15.uih.csv

```diff
@@ -49,6 +49,7 @@ r;wi,,EhpHeatBufferAvailable,EhpHeatBufferAvailable,,,,"5C00",,,onoff,,,heat buf
 [SW>=362]r,,reserve1,reserve 1,,,,"9D00",,,UCH,,,for future use
 # ##### HC Names and Tel No #####,,,,,,,,,,,,,
 r;w,,HcName2,aaucHcNames CH2,,,,"4108",,,STR:11,,,name of CH2
+r;w,,HcName3,aaucHcNames CH3,,,,"4110",,,STR:11,,,name of CH3
```

Comment:
please add it on the german version as well

Reply:
added to ebusd-2.1.x/de/vaillant/15.uih.csv